framework/src/ble_manager/ble_manager_lwnl_listener.c

```diff
@@ -97,10 +97,10 @@ int lwnl_fetch_ble_event(int fd, void *buf, int buflen)
 		_lwnl_call_event(fd, status, len);
 	} else {
 		/* Remove wifi data in socket. */
-		char *buf = (char *)malloc(len);
-		int nbytes = read(fd, (char *)buf, len);
-		BLE_LOG_VERBOSE("nbytes : %d / len : %d\n", nbytes, len);
-		free(buf);
+		char *t_buf = (char *)malloc(len);
+		nbytes = read(fd, (char *)t_buf, len);
+		BLE_LOG_INFO("read unused data nbytes : %d / len : %d\n", nbytes, len);
+		free(t_buf);
 		return;
```

Comment:
Please check the return type. The return type of this function is `int`, so it is better to return integer value.

Reply:
I fixed it